docs/joss_paper/paper.bib

```diff
@@ -118,7 +118,7 @@ @article{vercauteren2009diffeomorphic
 @article{brown2020introduction,
```

Comment:
Name not: (Brown, Nina Montana), replace with (Montana Brown, Nina).
Off the top of my head, these are names we have used prior: Shaheer U. Saeed (@s-sd) and Zachary M. C. Baum (@zacbaum ), check with auths.

Reply:
Thank you! I think if a change is possible, 'Shaheer U. Saeed' might be better